releasenotes/notes/add-trainable-fidelity-statevector-kernel-6d9c927d16c1cb35.yaml

```diff
@@ -0,0 +1,43 @@
+---
+features:
+  - |
+    Added a new :class:`~qiskit_machine_learning.kernels.TrainableFidelityStatevectorKernel` class
+    that provides a trainable version of
+    :class:`~qiskit_machine_learning.kernels.FidelityStatevectorKernel`.
+    This mirrors the relationship between
+    :class:`~qiskit_machine_learning.kernels.TrainableFidelityQuantumKernel` and
+    :class:`~qiskit_machine_learning.kernels.FidelityQuantumKernel`.
+    I.e., :class:`~qiskit_machine_learning.kernels.TrainableFidelityStatevectorKernel` inherits from
+    :class:`~qiskit_machine_learning.kernels.FidelityStatevectorKernel` and
+    :class:`~qiskit_machine_learning.kernels.TrainableKernel`.
+
+    Excepting the
+    :class:`~qiskit_machine_learning.kernels.TrainableFidelityStatevectorKernel`-specific
+    features, this is used with
+    :class:`~qiskit_machine_learning.kernels.algorithms.QuantumKernelTrainer`,
+    identically to
+    :class:`~qiskit_machine_learning.kernels.TrainableFidelityStatevectorKernel`.
+
+    Example usage snippet:
+
+    .. code-block:: python
+
+      # Instantiate trainable fidelity statevector kernel.
+      quant_kernel = TrainableFidelityStatevectorKernel(
+          feature_map=feature_map,
+          training_parameters=training_parameters,
+      )
+
+      # Set up the optimizer.
+      cb_qkt = QKTCallback()
+      spsa_opt = SPSA(maxiter=10, callback=cb_qkt.callback, learning_rate=0.05, perturbation=0.05)
```

Comment:
I had no idea what QKTCallback was, In searching in Qiskit ML docs I see its a class defined in one of the tutorials. As such I guess this sample you are thing its comparable to that code portion in 08_quantum_kernel_trainer tutorial? Maybe that should be made explicit here otherwise without that context its not easy to figure things. I guess SPSA was chosen likewise.

Reply:
Yes, that's a good point. I lifted this snippet straight from a modified script lifted from the tutorial that I was using to quickly check the performance. As the script and snippet work fine with the default settings, I have removed those lines from the release note.